src/main/java/com/onlinebookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package com.onlinebookstore.dto.user;
+
+import com.onlinebookstore.validation.Email;
+import com.onlinebookstore.validation.FieldMatch;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+@FieldMatch(
+        field = "password",
+        fieldMatch = "repeatPassword",
+        message = "Password and repeat password shouldn't be empty and should be equal"
+)
+public class UserRegistrationRequestDto {
+    @NotNull
+    @Size(min = 4, max = 50)
+    @Email
+    private String email;
```

Comment:
```suggestion
    @Email
    private String email;
```

Reply:
Yes, the condition is redundant, even in the absence of validator. Got it. Thank you!